### PR TITLE
added small suite of RNNG models

### PR DESCRIPTION
**Pull request by benlipkin**
This PR includes a small suite of RNNG models trained on a few different datasets at a few different parameter sizes.

### CODE REVIEW

File: brainscore_language/models/rnng/test.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    [
+        ("rnn-tdg-ptb", 512),
+        ("rnn-lcg-ptb", 512),
+        ("rnn-tdg-ptboanc", 512),
+        ("rnn-lcg-ptboanc", 512),
+        ("rnn-tdg-ptboanc-1024", 1024),
+        ("rnn-lcg-ptboanc-1024", 1024),
+    ],
+)
+def test_neural(model_identifier, feature_size):
```
Comment (mschrimpf) [inline brainscore_language/models/rnng/test.py:20]: are these models also capable of producing behavioral outputs?

**Review by mschrimpf — APPROVED** [review]

File: brainscore_language/models/rnng/test.py (hunk 1 above)
Comment (benlipkin) [reply brainscore_language/models/rnng/test.py:20]: in theory yes, but not under the current implementation. There's some non-trivial aspects about how the tasks are currently orchestrated from brainscore, and what the model would need to do to get token-level surprisals. I plan to return to this in the future, but need to set it aside for now for more pressing things.

Comment (mschrimpf) [reply brainscore_language/models/rnng/test.py:20]: ok sounds good! Will they just raise a `NotImplementedError` when asked to start behavior?

Comment (benlipkin) [reply brainscore_language/models/rnng/test.py:20]: Inside the container, a `NotImplementedError` is raised, and then it gets escalated by the `ContainerSubject` class here: https://github.com/brain-score/language/blob/benlipkin/rnng-models/brainscore_language/model_helpers/container.py#L183 which returns the full traceback from inside the container as well as a high-level error message

**Pull request merged**